addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationJob.java

```diff
@@ -123,6 +125,14 @@ public String getSummary() {
         return EMPTY_SUMMARY;
     }
 
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
```

Comment:
Add `setChanged()`.

Reply:
I called `jobChanged()`  directly  in `PlanTreeTableModel:: setValueAt()`, not sure if this is the right way :

```java
    @Override
    public void setValueAt(Object value, Object node, int column) {
        if (column == ENABLED_INDEX) {
            DefaultMutableTreeNode treeNode = (DefaultMutableTreeNode) node;
            if (!treeNode.isRoot()) {
                Object obj = treeNode.getUserObject();
                if (obj instanceof AutomationJob) {
                    AutomationJob job = (AutomationJob) obj;
                    job.setEnabled(((Boolean) value).booleanValue());
                    jobChanged(job);
                }
            }
        }
    }
```